recipes/eumdac/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "eumdac" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: bc320d6ca7ce9f192004de01663ffa91c98ac587638541eee80b5608b733efb8
+
+build:
```

```suggestion
build:
  noarch: python
  entry_points:
    - eumdac = eumdac.cli:cli
```

This seems to be a pure python package. As such, it can be [`noarch`](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds), which will make it installable on all platforms from a single build.
Entry points of type `console_script` should be listed here explicitly.